server/server.js

```diff
@@ -134,24 +134,31 @@ server.get('/api/pregnancy-centers/near-me', isLoggedInAPI, function (req, res)
 	})
 })
 
-server.get('/api/pregnancy-centers/verify', isLoggedInAPI, function (req, res) {
-
-	// We can change the search conditions in the future based on how recently the pregnancy center has been verified ...
-	// and what attributes were verified
+server.get('/api/pregnancy-centers/verify', isLoggedInAPI, async function (req, res) {
+	const pregnancyCenter = await PregnancyCenterModel.findOne({
+		'verified.address': null,
+	})
+	.lean()
 
-	log.info(req.user)
+	if (!pregnancyCenter) {
+		res.boom.notFound()
+	}
 
-	PregnancyCenterModel.findOne({'verified.address': null}, function (err, pregnancyCenterToVerify) {
-		if (err) {
-			log.error(err)
-			res.boom.badImplementation()
-		} else if (!pregnancyCenterToVerify) {
-			res.boom.notFound()
-		} else {
-			res.status(200).json(pregnancyCenterToVerify)
-		}
+	const primaryContact = pregnancyCenter.primaryContact
 
+	const user = await UserModel.findOne({
+		_id: primaryContact,
 	})
+	.lean()
```

Comment:
We need to get the info from the user collection since we're only storing the `_id` in the PregnancyCenter collection